examples/multi-container/ruby/README.md

```diff
@@ -0,0 +1,120 @@
+# Cloud SQL Auth Proxy Sidecar
+
+In the following example, we will deploy the Cloud SQL Proxy as a sidecar to an
+existing application which connects to a Cloud SQL instance.
+
+## Before you begin
+
+1. If you haven't already, [create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+
+1. Create a 2nd Gen Cloud SQL Instance by following these
+[instructions](https://cloud.google.com/sql/docs/postgres/create-instance).
+Note the connection string, database user, and database password that you create.
+
+1. Create a database for your application by following these
+[instructions](https://cloud.google.com/sql/docs/postgres/create-manage-databases).
+Note the database name.
+
+## Deploying the Application
+
+The application you will be deploying should connect to the Cloud SQL Proxy using
+TCP mode (for example, using the address "127.0.0.1:5432"). Follow the examples
+on the [Connect Auth Proxy documentation](https://cloud.google.com/sql/docs/postgres/connect-auth-proxy#expandable-1)
+page to correctly configure your application.
+
+The connection pool is configured in the following sample:
+
+```ruby
+require 'sinatra'
+require 'sequel'
+
+set :bind, '0.0.0.0'
+set :port, 8080
+
+# Configure a connection pool that connects to the proxy via TCP
+def connect_tcp
+    Sequel.connect(
+        adapter: 'postgres',
+        host: ENV["INSTANCE_HOST"],
+        port: ENV["DB_PORT"],
+        database: ENV["DB_NAME"],
+        user: ENV["DB_USER"],
+        password: ENV["DB_PASS"],
+        pool_timeout: 5,
+        max_connections: 5,
+    )
+end
+
+DB = connect_tcp()
+```
+
+Next, build the container image for the main application and deploy it:
+
+```bash
+gcloud builds submit --tag gcr.io/<YOUR_PROJECT_ID>/run-cloudsql
+```
+
+Finally, create a revision YAML file (multicontainers.yaml), using the `example.yaml`
+file as a reference for the deployment, listing the Cloud SQL container image as
+a sidecar:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  annotations:
+     run.googleapis.com/launch-stage: ALPHA
+  name: multicontainer-service
+spec:
+  template:
+    metadata:
+      annotations:
+        run.googleapis.com/execution-environment: gen1 #or gen2
```

Comment:
should we add the VPC Connector integration part here but have it commented out like we do `--private-ip`?

Reply:
I think that's a good idea